aws-lambda/todo-reminder/template.yaml

```diff
@@ -19,4 +19,10 @@ Resources:
       Environment: # More info about Env Vars: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#environment-object
         Variables:
           PARAM1: VALUE
-          ENV_NAME: !Ref EnvironmentName
\ No newline at end of file
+          ENV_NAME: !Ref EnvironmentName
+      Events:
+        ScheduledEvent:
+          Type: Schedule
+          Properties:
+            Schedule: cron(0/10 * * * ? *)
+            Enabled: True
```

Comment:
This is definitely a valid addition to the template. However, the focus of the article was primarily the implementation of the lambda. There are lots of ways this Lambda might be invoked periodically, and the article didn't go into that.

We _could_ add this change, and even update the article, but it's kind of off topic.

Reply:
When I was going through your article I was happy to read that we would code a function that runs every few minutes, because it was exactly what I was looking for. After studying and running the code I found out that it does not do that, so I came back to the article wondering whether I had missed or misunderstood something. So it was kind of confusing.
 The rest of the article was **awesome** and helped me lot! Many thanks, cheers, Chillbirdo